kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/clock/ClockEvent.java

```diff
@@ -23,6 +25,8 @@
     /** Topic of the ClockEvent */
     public static final String CLOCK_EVENT_TOPIC = "org/eclipse/kura/clock";
 
+    public static final ClockEvent EMPTY = new ClockEvent(Collections.<String, Object> emptyMap());
```

Comment:
@ctron Analyzing the api changes I went through this.
Why we need this EMPTY element in the APIs? Being public, this implies a change on the clock package version in manifest file. 
Furthermore, I don't see a big advantage in having it here. Maybe it's better to preserve the api and move this definition in the impl class. What do you think?